job/job.go

```diff
@@ -34,6 +37,20 @@ type Job struct {
 	instance *packet.Instance
 }
 
+// isHardwareHavingActiveWorkflow fetches workflows for a given hwID and returns
+// the status true if there is a pending (active) workflow for hwID
+// hwID is the hardware/worker ID corresponding to the MAC
+func isHardwareHavingActiveWorkflow(wcl *tw.WorkflowContextList) (bool, error) {
```

Comment:
let rename this to `hasActiveWorkflow` its shorter and easier to grok.

Reply:
done.